### PR TITLE
Testing PSI selection

### LINKED ISSUE #71: Testing PSI selection
# Description
Write tests for PSI selection (classes, methods and lines). Verify that only the classes that are supposed to be selected are selected, only the methods that are supposed to be selected are selected and only the lines that are supposed to be selected are selected.

# Definition of Done
* [x]  Class selection is tested
* [x]  Only the allowed classes are selected
* [x]  Method selection is tested
* [x]  Only the allowed methods are selected
* [x]  Line selection is tested
* [x]  Only the allowed lines are selected

### PR DESCRIPTION
**Pull request by jzelenjak**
# Description of changes made
Testing for Class, Method and Line Selection (PSI) has been added

# Why is merge request needed
This merge request is needed to ensure that our PSI selection is working as intended 

# Other notes
Closes #71 

# What is missing?

- [x] I have checked that I am merging into correct branch

### CODE REVIEW

File: src/test/resources/project/untitled/src/main/java/ArrayUtils.java [hunk 1]
```diff
@@ -1,5 +1,11 @@
 /**
- * (From Apache Commons Lang: https://github.com/apache/commons-lang/blob/d2687419c6973572d1621afc1b8546f5262769c3/src/main/java/org/apache/commons/lang3/ArrayUtils.java#L2325)
+ * (From Apache Commons Lang: https://github.com/apache/commons-lang/blob/d2687419c6973572d1621afc1b8546f5262769c3/src/main/java/org/apache/commons/lang3/
```
Comment (kirilvasilev16) [inline src/test/resources/project/untitled/src/main/java/ArrayUtils.java]: Are the changes necessary here?

Comment (jzelenjak) [reply src/test/resources/project/untitled/src/main/java/ArrayUtils.java]: No, I did not even know they were there until you told me. I have fixed it and now they are as they were before.

**Review by kirilvasilev16 — APPROVED** [review]
LGTM

**Review by bolkhod — APPROVED** [review]
Looks Good To Me

**Pull request merged**